forte/pymodule.py

```diff
@@ -674,6 +694,10 @@ def gradient_forte(name, **kwargs):
                                            mo_space_info, ints)
         energy = casscf.compute_energy()
 
+    if correlation_solver == 'DSRG-MRPT2':
```

Comment:
```suggestion
    if job_type == 'NEWDRIVER' and correlation_solver == 'DSRG-MRPT2':
```

Reply:
It used to be like the way you wrote, but I just thought the first condition was redundant.

Reply:
You should probably test the following works as intended. Suppose I set `job_type casscf`, CASSCF will compute gradient and write bunch of densities. Do we need this extra computation? Of course it will be minimal time compared to the PT2 gradients.

Reply:
In fact, the CASSCF gradient contribution is considered in the dsrggradient code by default.

Reply:
and thus the previous casscf gradient code is not called